From 08cbbda5b4c0a52965ac5ba008c1de4f241c2abd Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.org>
Date: Tue, 18 Jun 2019 11:19:59 +0100
Subject: [PATCH 668/773] overlays: Cosmetic change to upstream overlay

The dwc2 overlay no longer uses the dwc2_usb label, and the latest
ovmerge (which generates the upstream overlay) removes unused labels.
Update the checked-in version to match.

Signed-off-by: Phil Elwell <phil@raspberrypi.org>
---
 arch/arm/boot/dts/overlays/upstream-overlay.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/arm/boot/dts/overlays/upstream-overlay.dts
+++ b/arch/arm/boot/dts/overlays/upstream-overlay.dts
@@ -113,7 +113,7 @@
 		target = <&usb>;
 		#address-cells = <1>;
 		#size-cells = <1>;
-		dwc2_usb: __overlay__ {
+		__overlay__ {
 			compatible = "brcm,bcm2835-usb";
 			dr_mode = "otg";
 			g-np-tx-fifo-size = <32>;
